src/middleware/language.js

```diff
@@ -52,16 +61,19 @@ function setup({ languages = [] } = {}) {
     const acceptLanguage = req.get('Accept-Language');
 
     if (!acceptLanguage) {
-      req.language = languages[0];
+      req.language = defaultLanguage;
```

Comment:
Since the languages array is ordered by priority, in which case will the default language not be the first of the array?

Reply:
It was non obvious to me that the default language was the first one.
The array of `authorized languages` is used to find a valid language from the user accepted languages. Its order is not big deal.
What is most important is the order of the user accepted languages.